octoprint_slic3r/__init__.py

```diff
@@ -166,7 +166,7 @@ def get_slicer_properties(self):
 	def get_slicer_default_profile(self):
 		path = self._settings.get(["default_profile"])
 		if not path:
-			path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "profiles", "default.profile.ini")
+			path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "static/profiles", "default.profile.ini")
```

Comment:
Well, do I get it right, that, locating profiles in profiles not in static/profiles should be better?